sonar-scanner-engine/src/test/java/org/sonar/scanner/mediumtest/issuesmode/ScanOnlyChangedTest.java

```diff
@@ -47,25 +55,34 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@RunWith(Parameterized.class)
 public class ScanOnlyChangedTest {
-  @org.junit.Rule
+  private static SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yyyy");
+
+  /*
+   * Run with branch too to reproduce SONAR-9141 and SONAR-9160
+   */
+  @Parameters(name = "branch enabled:{0} projectKey: {1}")
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][] {
+      {false, "sample"}, {true, "sample:branch"}
```

Comment:
Maybe it could be interesting to test a key having ":" to be sure there is no remains of getting the branch by parsing the key.